user-guide/modules/ROOT/pages/advanced-builds.adoc

```diff
@@ -1,3 +1,247 @@
 = Advanced Builds
+:navtitle: Advanced Builds
+
+This section covers a number of advanced build scenarios that you may come across if your project has complex, legacy, or non-standard, components or processes.
+
+== Build with a Non-default Compiler
+
+Using the `b2` build system, you can specify the compiler you want to use with the `toolset` option. For example, if you want to use the GNU Compiler Collection (GCC), you would specify `toolset=gcc`. If you want to use Clang, you would use `toolset=clang`. For example, to build the Boost libraries with gcc:
+
+[source,txt]
```

Comment:
These are not `txt`.

Reply:
agreed, removed

Reply:
Have a look at other options. I think this is `bash`.

See https://github.com/cppalliance/mrdox/blob/07471c21416f0332d89c5260729bf86a914ca240/docs/modules/ROOT/pages/usage.adoc?plain=1